docs/Contributing.mdx

```diff
@@ -13,3 +13,10 @@ So, from the bottom of our hearts, _thank you_ 💕.
 
 Please read our [Code of Conduct](https://github.com/typescript-eslint/typescript-eslint/blob/main/CODE_OF_CONDUCT.md) first.
 If you're new to open source, you may also find the [How to Contribute to Open Source](https://opensource.guide/how-to-contribute) guide from https://opensource.guide useful.
+
+## More
```

Comment:
I think what I'd been hoping for in the issue was more like a "next steps" suggestion. Right now the list is mostly a rote description of what's already in the sidebar, with just a bit more info on each.

Suggestion: let's change the heading...

```suggestion
## Next Steps
```

...and have the steps the users take be what @skondrashov suggested in the issue:

1. Finding or opening an issue
2. Setting up a local environment
3. Making a PR

Thoughts?

Reply:
i agree. i misleading the issue

I modified it to reflect the above content, but I am confused as to whether it was modified as intended. Thank you for checking it out! If there is anything strange, please tell me.